.github/workflows/tests.yml

```diff
@@ -66,4 +66,4 @@ jobs:
 
       - name: Run pytest
         shell: bash
-        run: poetry run pytest -q tests
+        run: poetry run pytest -q tests
```

Comment:
There seem to be some whitespace changes in this latest iteration. Namely, missing EOF newlines. Perhaps something VS Code is doing? I suggest reviewing the PR file list in GitHub to ensure there isn't any unintended whitespace changes: https://github.com/python-poetry/poetry-core/pull/40/files

Reply:
Yeah, I just realized it, goddamn VS Code man 🙂  I specifically used "Save Without Formatting" option, and it still removed the `\n`. I'll try to fix it.

Reply:
Hehe. Helpful editors being helpful. 😉

There are a few other missing newlines in the test fixtures. Probably worth addressing unless they need to be absent for some reason.

Reply:
Added a new line to the end of test_builders.py

Reply:
Nvm, if I add one, the linter fails. I guess I'll keep it as is.